### PR TITLE
MON-24120 HA installation 4 nodes 23.10

### PR DESCRIPTION
**Pull request by thearchit3ct**
## Description

Please include a short summary of the changes and what is the purpose of the PR. Any relevant information should be added to help reviewers.

## Target version (i.e. version that this PR changes)

- [ ] 21.10.x
- [ ] 22.04.x
- [ ] 22.10.x
- [ ] 23.04.x
- [x] 23.10.x
- [ ] 24.04.x
- [ ] Cloud
- [ ] Monitoring Connectors

### CODE REVIEW

File: i18n/fr/docusaurus-plugin-content-docs/version-23.10/installation/installation-of-centreon-ha/installation-4-nodes.md [hunk 1]
```diff
@@ -904,6 +1081,43 @@ pcs qdevice setup model net --enable --start
 pcs qdevice status net --full
 ```
 
+</TabItem>
+<TabItem value="Alma Linux 9" label="Alma Linux 9">
+
+```bash
+dnf config-manager --enable ha
```
Comment (itoussies) [inline i18n/fr/docusaurus-plugin-content-docs/version-23.10/installation/installation-of-centreon-ha/installation-4-nodes.md:1088]:
```suggestion
dnf config-manager --enable highavailability
```

**Review by itoussies — CHANGES_REQUESTED** [review]

File: i18n/fr/docusaurus-plugin-content-docs/version-23.10/installation/installation-of-centreon-ha/installation-4-nodes.md [hunk 2]
```diff
@@ -1073,6 +1323,24 @@ pcs resource create "ms_mysql" \
...
+
+```bash
+pcs resource create "ms_mysql" \
+    ocf:heartbeat:mariadb-centreon \
+    config="/etc/my.cnf.d/mariadb-server.cnf" \
+    pid="/var/lib/mysql/mysql.pid" \
+    datadir="/var/lib/mysql" \
+    socket="/var/lib/mysql/mysql.sock" \
+    binary="/usr/bin/mysqld_safe" \
+    node_list="@CENTRAL_MASTER_NAME@ @CENTRAL_SLAVE_NAME@" \
```
Comment (omercier) [inline i18n/fr/docusaurus-plugin-content-docs/version-23.10/installation/installation-of-centreon-ha/installation-4-nodes.md:1336]:
```suggestion
    node_list="@DATABASE_MASTER_NAME@ @DATABASE_SLAVE_NAME@" \
```

Comment (omercier) [reply i18n/fr/docusaurus-plugin-content-docs/version-23.10/installation/installation-of-centreon-ha/installation-4-nodes.md:1336]: Cf MON-32722 and please update it in all other occurrences of node_list definition

**Review by omercier — CHANGES_REQUESTED** [review]

File: i18n/fr/docusaurus-plugin-content-docs/version-23.10/installation/installation-of-centreon-ha/installation-4-nodes.md (hunk 2 above)
Comment (omercier) [reply i18n/fr/docusaurus-plugin-content-docs/version-23.10/installation/installation-of-centreon-ha/installation-4-nodes.md:1336]: this command is valid for 2 nodes clusters only (where mysql is running on the centrals)

**Review by omercier — DISMISSED** [review]

**Review by omercier — APPROVED** [review]

**Review by smau-centreon — DISMISSED** [review]

**Review by cedricmeschin — APPROVED** [review]

**Pull request merged**